modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/js/Tabs.js

```diff
@@ -78,57 +78,73 @@ export default function ({
 	return (
 		<>
 			<ClayTabs className="nav-left" modern>
-				<ClayTabs.Item
-					active={activeTabKeyValue === 0}
-					innerProps={{
-						'aria-controls': 'tabpanel-1',
-					}}
-					onClick={() => {
-						display(navDescriptionId);
-						setActiveTabKeyValue(0);
-					}}
-				>
-					{Liferay.Language.get('full-description')}
-				</ClayTabs.Item>
+				{hasDescription ? (
+					<ClayTabs.Item
+						active={activeTabKeyValue === 0}
+						innerProps={{
+							'aria-controls': 'tabpanel-1',
+						}}
+						onClick={() => {
+							display(navDescriptionId);
+							setActiveTabKeyValue(0);
+						}}
+					>
+						{Liferay.Language.get('full-description')}
+					</ClayTabs.Item>
+				) : (
+					<></>
+				)}
 
-				<ClayTabs.Item
-					active={activeTabKeyValue === 1}
-					innerProps={{
-						'aria-controls': 'tabpanel-2',
-					}}
-					onClick={() => {
-						display(navSpecificationsId);
-						setActiveTabKeyValue(1);
-					}}
-				>
-					{Liferay.Language.get('specifications')}
-				</ClayTabs.Item>
+				{hasCPDefinitionSpecificationOptionValues ? (
+					<ClayTabs.Item
+						active={activeTabKeyValue === 1}
+						innerProps={{
+							'aria-controls': 'tabpanel-2',
+						}}
+						onClick={() => {
+							display(navSpecificationsId);
+							setActiveTabKeyValue(1);
+						}}
+					>
+						{Liferay.Language.get('specifications')}
+					</ClayTabs.Item>
+				) : (
+					<></>
```

Comment:
this slack chat https://liferay.slack.com/archives/CNBG06JS3/p1667423001499459?thread_ts=1667279464.808489&cid=CNBG06JS3 made me change my mind on these changes

Reply:
@andrea-ale-sbarra , did you want me to make additional changes? Also, is there a use-case in approving/rejecting from the mini-cart?I I personally still do not see why we cannot just redirect to pending commerce orders page always.

Reply:
No I don't think we need further changes on this.
I didn't understand questions about minicart and pending order, I'm missing the context sorry.

Reply:
hey @andrea-ale-sbarra , we could talk about it next week in-person. Maybe it would be better since it will change how our workflow works.